modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -201,7 +201,7 @@ component accessors="true" singleton threadSafe{
 	function siteBaseURL(){
 		return replacenocase( 
 			getRequestContext().buildLink( linkTo='', ssl=getRequestContext().isSSL() ), 
-			"index.cfm", 
+			"index.cfm/", 

```

Comment:
It is an issue, this is a fix.
You show me a place where having 2 slashes is a good thing. 

Build Link adds the / on the end, so that replace leaves double slashes.